packages/fondue/package.json

```diff
@@ -63,7 +63,7 @@
         "test:browser": "cypress open --component",
         "test:file": "cypress run --component --spec",
         "typecheck": "tsc --noEmit",
-        "storybook": "storybook dev -p 6006",
+        "storybook": "storybook dev -p 6007",
```

Comment:
what is the reason to switch the ports between the components and legacy fondue?